components/AppNav.tsx

```diff
@@ -17,47 +17,47 @@ type AuthLinkProps = {
   session: any
 }
 
+type NavItem = NavLinkProps & { name: string }
+
 const dropdownMenuItems = [
   { title: 'Lessons', path: '/admin/lessons' },
   { title: 'Users', path: '/admin/users' },
   { title: 'Alerts', path: '/admin/alerts' }
 ]
+const navItems: NavItem[] = [
+  { path: '/curriculum', name: 'Curriculum' },
+  {
+    path: 'https://github.com/garageScript/c0d3-app',
+    name: 'Repo',
+    external: true
+  },
+  {
+    path:
+      'https://www.notion.so/Table-of-Contents-a83980f81560429faca3821a9af8a5e2',
+    name: 'Journey',
+    external: true
+  },
+  { path: 'https://chat.c0d3.com', name: 'Help', external: true },
+  { path: '/contributors', name: 'Contributors' }
+]
 
 const NavBar: React.FC<AuthLinkProps> = ({ session }) => {
   const isAdmin = _.get(session, 'user.isAdmin', '')
+  const location = window && '/' + window.location.pathname.split('/')[1]
+
   return (
     <div className="navbar-nav collapse navbar-collapse">
-      <NavLink
-        path="/curriculum"
-        activePath="/curriculum"
-        className="nav-item nav-link"
-      >
-        Curriculum
-      </NavLink>
-      <NavLink
-        path="https://github.com/garageScript/c0d3-app"
-        className="nav-item nav-link"
-        external
-      >
-        Repo
-      </NavLink>
-      <NavLink
-        path="https://www.notion.so/Table-of-Contents-a83980f81560429faca3821a9af8a5e2"
-        className="nav-item nav-link"
-        external
-      >
-        Journey
-      </NavLink>
-      <NavLink
-        path="https://chat.c0d3.com"
-        className="nav-item nav-link"
-        external
-      >
-        Help
-      </NavLink>
-      <NavLink path="/contributors" className="nav-item nav-link">
-        Contributors
-      </NavLink>
+      {navItems.map(button => (
```

Comment:
You should destructure the button object here, that will make the code more concise

Reply:
Changed.